samples/external-ds/prep-ds.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+
+unzip /opt/DS-7.5.1.zip
```

Comment:
Can you make this a variable so folks can easily use this with whatever version?

ZIP_PATH=${1:-/opt/DS-7.5.1.zip}

Reply:
Actually I think I'm going to add a comment in the doc for the user to edit the script as required rather than start to provide arguments for these scripts.